crates/web-plugins/didcomm-messaging/did-utils/src/didkit.rs

```diff
@@ -67,13 +69,13 @@ impl Document {
             context,
             also_known_as: None,
             controller: None,
-            authentication: None,
-            assertion_method: None,
+            authentication: Some(vec![]),
+            assertion_method: Some(vec![]),
             capability_delegation: None,
             capability_invocation: None,
-            key_agreement: None,
-            verification_method: None,
-            service: None,
+            key_agreement: Some(vec![]),
+            verification_method: Some(vec![]),
+            service: Some(vec![]),
```

Comment:
Including these fields in the DID document is somewhat necessary. Many sections of the codebase validate the DID document (including empty documents), so these fields are almost always checked; otherwise, the validations would fail. This also helps prevent deserialization and serialization errors during the conversion from document representations. Without this, we would need to implement these fields across all code sections that require document validation, tracking all of these would be a pain.